act/api/fact.py

```diff
@@ -299,13 +299,16 @@ class Fact(ActBase):
               serializer=lambda fact_type: fact_type.name),
         Field("value", default=""),
         Field("id", serializer=False),
-        # For now, do not serialize/deserialize source
         Field("flags", serializer=False),
-        Field("source", deserializer=False, serializer=False),
+        Field("origin", deserializer=Origin),
+        Field("added_by", deserializer=Origin, serializer=False),
+        Field("trust", serializer=False),
+        Field("confidence"),
+        Field("certainty", serializer=False),
         Field("timestamp", serializer=False),
         Field("last_seen_timestamp", serializer=False),
         Field("in_reference_to", deserializer=ReferencedFact),
-        Field("organization", deserializer=Organization, serializer=False),
+        Field("organization", deserializer=Organization),
```

Comment:
Organization can be specified in the fact and added to the platform, so it should be serialized